add_wasm.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"syscall/js"
+)
+
+// Add two numbers.
+func Add(a int, b int) int {
+	return a + b
+}
+
+// Gets a HTML element given an ID.
+func getElementById(id string) js.Value {
+	return js.Global().Get("document").Call("getElementById", id)
+}
+
+// Gets a numeric value from an HTML element body given an ID.
+func getNumberFromElement(id string) (int, error) {
+	element := getElementById(id)
+	valueStr := element.Get("value").String()
+	return strconv.Atoi(valueStr)
+}
+
+// Adds the two input elements and sets the sum to another element.
+func AddAndSetResult() {
```

Comment:
minor: why is `Add`, `AddAndSetResult` capitalized while others aren't ?

Reply:
Per convention, internal methods that won't be imported start with a lower case (`addAndSetResult` etc).
But methods that will be imported outside the file (`Add`) will start from upper case.

Have changed `AddAndSetResult ` -> `addAndSetResult `

Reply:
Good to know. 👍 
Thanks for updating.